packages/sds/src/sds.ts

```diff
@@ -203,13 +205,51 @@ export class MessageChannel {
     );
   }
 
+  /**
+   * Send a sync message to the SDS channel.
+   *
+   * Increments the lamport timestamp, constructs a `Message` object
+   * with an empty load. Skips outgoing buffer, filter, and local log.
+   *
+   * See https://rfc.vac.dev/vac/raw/sds/#send-sync-message
+   *
+   * @param callback - A callback function that returns a boolean indicating whether the message was sent successfully.
+   */
+  public sendSyncMessage(callback?: (message: Message) => boolean): boolean {
+    this.lamportTimestamp++;
+
+    const emptyMessage = new Uint8Array();
+
+    const message: Message = {
+      messageId: MessageChannel.getMessageId(emptyMessage),
+      channelId: this.channelId,
+      lamportTimestamp: this.lamportTimestamp,
+      causalHistory: this.messageIdLog
+        .slice(-this.causalHistorySize)
+        .map(({ messageId }) => messageId),
+      bloomFilter: this.filter.toBytes(),
+      content: emptyMessage
+    };
+
+    if (callback) {
+      return callback(message);
```

Comment:
is thins callback intended to be used with `async` operations?

I might misunderstand something but I believe it is intended to be used like:
```typescript
node.sds.sendSyncMessage(async () => {
  const resp = await  node.lightPush.send();
  if (resp.success) return true;
  return false;
```

Reply:
good point, also curious

another point: this function isnt really "sending", but "preparing" the message, to be sent with lightpush, right?

perhaps doing:
```ts
public createSyncMessage(): Message { ... }

const msg = channel.createSyncMessage();
await node.lightPush.send(msg);
```

or if we want to keep the same API, perhaps:
```ts
withSyncMessage(callback)
```